doc/introduction/templates.rst

```diff
@@ -179,15 +158,13 @@ Tensor networks
 Tensor-network templates create quantum circuit architectures where circuit blocks
 can be broadcast with the shape and connectivity of tensor networks.
 
-.. customgalleryitem::
-    :link: ../code/api/pennylane.MPS.html
-    :description: Matrix Product State
-    :figure: ../_static/templates/tensornetworks/MPS_template.png
+.. gallery-item::
+    :description: :doc:`Matrix Product State <../code/api/pennylane.MPS>`
+    :figure: _static/templates/tensornetworks/MPS_template.png
 
-.. customgalleryitem::
-    :link: ../code/api/pennylane.TTN.html
-    :description: Tree Tensor Network
-    :figure: ../_static/templates/tensornetworks/TTN_template.png
+.. gallery-item::
+    :description: :doc:`Tree Tensor Network <../code/api/pennylane.TTN>`
+    :figure: _static/templates/tensornetworks/TTN_template.png
 
 .. raw:: html
 
```

Comment:
The sphinx doc check seems to be emitting a warning here?

![image](https://user-images.githubusercontent.com/2959003/163741501-00625e89-4dbc-4ccc-8936-44ad10e3d695.png)

Reply:
Hmm, that's strange. All of the code has been updated to use the new `.. gallery-item` directive instead. 🤔

Reply:
oh if the action is messing up we can ignore it. (Maybe it's a bug in github's annotation API 🤔)

Reply:
Everything is working now! This might have been caused by a divergence between PennyLane and my fork.